cypress/e2e/local/sampleForSingleEligibleApplicant.cy.ts

```diff
@@ -0,0 +1,258 @@
+import { faker } from '@faker-js/faker/locale/en_GB';
+import HomePage from '../../pages/home';
+import SignInPage from '../../pages/signIn';
+import VerifyPage from '../../pages/verify';
+import StartPage from '../../pages/start';
+import {
+  generateEmailAddress,
+  getRandomGender,
+  TitleEnum,
+} from '../../../testUtils/personHelper';
+import AgreeTermsPage from '../../pages/agreeTerms';
+import ApplyHouseholdPage from '../../pages/household';
+import ApplyExpectPage from '../../pages/apply/expect';
+import ApplyResidentIndexPage from '../../pages/apply/[resident]';
+import ApplyResidentPersonalDetailsPage from '../../pages/apply/[resident]/personal-details';
+import ApplyResidentSectionPage from '../../pages/apply/[resident]/[section]';
+import ApplyResidentAddressHistoryPage from '../../pages/apply/[resident]/address-history';
+import ApplyResidentCurrentAccommodationPage from '../../pages/apply/[resident]/current-accommodation';
+import ApplyResidentYourSituationPage from '../../pages/apply/[resident]/your-situation';
+import ApplyResidentSummaryPage from '../../pages/apply/[resident]/summary';
+import DeclarationPage from '../../pages/declaration';
+
+//ensure eligibility: over 55 and alone
+const birthDate = faker.date.birthdate({ mode: 'age', min: 56, max: 90 });
+const title = faker.helpers.enumValue(TitleEnum);
+const phoneNumber = faker.phone.number();
+const mainApplicantFirstName = faker.person.firstName();
+const mainApplicantLastName = faker.person.lastName();
+const postcode = 'A1 2BC';
+const email = generateEmailAddress();
+const applicationId = faker.string.uuid();
+const verificationCode = faker.number
+  .int({ min: 100000, max: 999999 })
+  .toString();
+
+const fillInTheSignUpForm = () => {
+  StartPage.getTitleDropdown().select(title);
+  StartPage.getFirstNameInput().type(mainApplicantFirstName);
+  StartPage.getLastNameInput().type(mainApplicantLastName);
+  StartPage.getDoBDayInput().type(birthDate.getDate().toString());
+  StartPage.getDoBMonthInput().type((birthDate.getMonth() + 1).toString());
+  StartPage.getDoBYearInput().type(birthDate.getFullYear().toString());
+  StartPage.getGenderOptions().check(getRandomGender());
+  StartPage.getNationalInsuranceNumberInput().type(
+    faker.string.alphanumeric(9)
+  );
+  StartPage.getPhoneNumberInput().type(phoneNumber);
+};
+
+const addressSearchAPIResponse = {
+  body: {
+    address: [
+      {
+        line1: 'TEST ADDRESS',
+        line2: '1 STREET',
+        line3: 'LOCAL',
+        line4: '',
+        town: 'CITY',
+        postcode: `${postcode}`,
+        UPRN: 11111111111,
+      },
+    ],
+    page_count: 1,
+    total_count: 1,
+  },
+};
+
+Cypress._.times(1, () => {
```

Comment:
Never seen this before, what is this for?

Couldn't we just start with the `describe` block?

Reply:
This way you can run the tests in that block as many times as you like back to back. Sometimes handy for troubleshooting if things seem a bit flaky.